src/aro/Diagnostics.zig

```diff
@@ -21,6 +21,10 @@ pub const Message = struct {
 
     pub const Extra = union {
         str: []const u8,
+        record_member: struct {
```

Comment:
This makes the `Extra` union 32 bytes instead of 16 (on a 64-bit system). We should use something like what is done in `errDeprecated` or `valueChangedStr` - write the string to `p.strings`, dupe it, and then just use the `.str` Extra field

Reply:
Done